engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Try
   *
   * @tparam K CallKey subtype
   */
-sealed trait JobDescriptor[K <: JobKey] {
+sealed trait JobDescriptor[K <: CallKey] {
```

Comment:
@sleongmgi If you send me your email address (either here or email me - my address is on my profile) I can send you a doc which goes into more detail on the new pluggable backend architecture which might be helpful